config/config.go

```diff
@@ -104,6 +117,9 @@ type SigningProfile struct {
 	NameWhitelist               *regexp.Regexp
 	ExtensionWhitelist          map[string]bool
 	ClientProvidesSerialNumbers bool
+	// IgnoredLintsMap is a bool map created from IgnoredLints when the profile is
+	// loaded. It facilitates set membership testing.
+	IgnoredLintsMap map[string]bool
```

Comment:
This should probably be unexported since it's an internal implementation detail, right?

Reply:
It needs to be exported to be used by the `signer/local` package. I could move the map out of the profile config to an unexported field of the `local.Signer` type alongside `lintPriv` but it will be more complicated because unlike the private key there can be multiple profiles per `Signer` and each can have its own configured ignore map. Overall I'd prefer to keep the map in the `SigningProfile` as an exported field. This also matches how other constructed fields are handled in `SigningProfile.populate`.